kalite/settings/base.py

```diff
@@ -248,9 +255,18 @@
         "OPTIONS": {
             "timeout": 60,
         },
+    },
+    "topic_tools": {
+        "ENGINE": "django.db.backends.sqlite3",
+        "NAME": TT_DATABASE_PATH,
+        "OPTIONS": {
+            "timeout": 60,
```

Comment:
Sets cache invalidation timeout: https://docs.djangoproject.com/en/1.5/ref/settings/#timeout

I think it's probably safe to remove from `topic_tools` database at least.